lib/tracing.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+// Copyright (C) 2020 SignalFx, Inc. All rights reserved.
+
+var tracing;
+
+try {
+  tracing = require('signalfx-tracing');
+} catch (err) {
+  tracing = {};
```

Comment:
I am wondering if perhaps we may want to let user know (with log statement) that we weren't able to get signalfx-tracing.